joomla/joomla-cms
# [3.10] Dont show the pre upgrade checker when the upgrade is not supported

### Summary of Changes

Dont show the pre upgrade checker when the upgrade is not supported

### Testing Instructions

- install 3.10 on a non supported setup for J4 that is checked by the updater lib (ex. mysql version older than 5.6)
- switch the update server to Joomla Next
- you see a pre upgrade checker
- apply this patch
- The pre upgrade checker is gone

#### Tip

when you dont have a setup with such an old mysql version you can fake it by adding `$dbMatch = false;` after this line here: https://github.com/joomla/joomla-cms/blob/3.10-dev/libraries/src/Updater/Update.php#L403

### Actual result BEFORE applying this Pull Request

You see a pre upgrade checker with false information as the update is not fully aviable nor working at all.
![image](https://user-images.githubusercontent.com/2596554/146040180-c39abe57-126b-4c65-bcd6-8615d0f29221.png)


### Expected result AFTER applying this Pull Request

The pre upgrade checker is gone
![image](https://user-images.githubusercontent.com/2596554/146039942-5b9939a0-993f-40a2-9821-cfab220652e8.png)


### Documentation Changes Required

none

cc @tecpromotion
